app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Admin::UsersController < Admin::BaseController
+  before_action :set_user, only: [:enable_or_disable_user, :promote_to_manager_or_demote_to_user]
+  
+  def index
+    @users = User.all_users_except_admin
+  end
+  
+  def enable_or_disable_user
+    if @user.disabled?
+      @user.enabled!
+      respond_to do |format|
+        format.html { redirect_to admin_root_path, notice: 'User Successfully Enabled' }
+        format.js
+      end
+    else
+      @user.disabled!
+      respond_to do |format|
+        format.html { redirect_to admin_root_path, notice: 'User Successfully Disabled' }
+        format.js
+      end
+    end
+  end
+  
+  def promote_to_manager_or_demote_to_user
```

Comment:
Please also refactor this action method as per this suggestion (https://github.com/farzam-azhar/pms/pull/1/files#r203999305).

Reply:
Done Sir.